components/utils.py

```diff
@@ -12,6 +12,7 @@
 import urllib
 import urllib2
 import hashlib
+import requests
```

Comment:
Hey. So.. you can't use libraries which are not part of the standard library as you're running scripts on a clean machine. If this worked for you, it's because of a bug we discovered like two days ago.